addons/web_editor/static/lib/odoo-editor/test/spec/format.test.js

```diff
@@ -13,177 +12,275 @@ const underline = async editor => {
 const strikeThrough = async editor => {
     await editor.execCommand('strikeThrough');
 };
+const setFontSize = size => editor => editor.execCommand('setFontSize', size);
 
 describe('Format', () => {
-    const b = (content, zws) => `<span${zws === 'first' ? ' oe-zws-empty-inline=""' : ''} style="font-weight: bolder;"${zws === 'last' ? ' oe-zws-empty-inline=""' : ''}>${content}</span>`;
-    const notB = (content, weight, zws) => `<span${zws === 'first' ? ' oe-zws-empty-inline=""' : ''} style="font-weight: ${weight || 'normal'};"${zws === 'last' ? ' oe-zws-empty-inline=""' : ''}>${content}</span>`;
+    const getZwsTag = (tagName, {style} = {}) => {
+        const styleAttr = style ? ` style="${style}"` : '';
+        return (content, zws) => {
+            const zwsFirstAttr = zws === 'first' ? ' oe-zws-empty-inline=""' : '';
+            const zwsLastAttr = zws === 'last' ? ' oe-zws-empty-inline=""' : '';
+            return `<${tagName}${zwsFirstAttr}${styleAttr}${zwsLastAttr}>${content}</${tagName}>`
+        };
+    }
+
+    const execWith = (functions) => {
+        return async (cb) => {
+            for (const fn of functions) {
+                await cb(fn);
+            }
+        }
+    }
+
+    const span = getZwsTag('span');
+
+    const strong = getZwsTag('strong');
+    const notStrong = getZwsTag('span', {style: 'font-weight: normal;'});
+    const spanBold = getZwsTag('span', {style: 'font-weight: bolder;'});
+    const b = (content) => `<b>${content}</b>`;
```

Comment:
I'm confused. Why is `b` treated differently than `strong` ? One gets a special "zws" treatment but not the other.

Reply:
It was a mistake. Good catch.